qa/restart-upgrade/build.gradle

```diff
@@ -51,7 +51,7 @@ testClusters {
         }
 
         // Skip test if version is 1.0, 1.1, 1.2 or 1.3 as they are not supported in those versions
-        if (knn_bwc_version.startsWith("1.")) {
+        if (knn_bwc_version.startsWith("1.") || knn_bwc_version.startsWith("2.0")) {
```

Comment:
Why not replace "1." with "2."?

Reply:
We are also running BWC Tests from 1.x versions to 2.x versions. So, we need to add that condition

Reply:
My understanding is we need both. opensearch_version defines current version, knn_bwc_version defines the version we want to be compatible with, so setting those to let's say 2.1.0 and 1.3.0 we testing if 2.1.0 knn is backward compatible with 1.3.0.
We are checking such compatibility for all versions defined [here](https://github.com/opensearch-project/k-NN/blob/main/.github/workflows/backwards_compatibility_tests_workflow.yml#L17) https://github.com/opensearch-project/k-NN/actions/runs/2398778347

Reply:
@VijayanB does it make sense? Actually we've added rolling upgrade bwc for both 1.3.2 and 2.0.0